easybuild/tools/utilities.py

```diff
@@ -59,7 +59,7 @@ def flatten(lst):
     return res
 
 
-def quote_str(val, escape_newline=False, prefer_single_quotes=False):
+def quote_str(val, escape_newline=False, prefer_single_quotes=False, syntax=None):
```

Comment:
@ocaisa Please add a `:param` line to the docstring for the new `syntax` parameter (and mention `Tcl` explicitly, since that's actually the only value that matters).

With that in mind, maybe you should be adding a `tcl=False` option instead, rather than checking for a magic value (and then use `tcl=self.SYNTAX == TCL` in `module_generator.py`)?